sitestats/sitestats-bundle/src/resources/Messages.properties

```diff
@@ -325,3 +325,6 @@ predefined_report5_description = Show users who have never visited the site.
 #
 predefined_report6_title = Users with no activity
 predefined_report6_description = Show users with no activity in site.
+
+format_datetime= MM/dd/yyyy hh:mm a
```

Comment:
Even having it in a util is better than doing this, like profile2 does. Though it would be nice if it was just a component or model all wicket tools shared. https://github.com/sakaiproject/sakai/blob/master/profile2/util/src/java/org/sakaiproject/profile2/util/ProfileUtils.java